content/en/monitors/types/process.md

```diff
@@ -67,13 +69,15 @@ If you cannot scope processes down to the granularity you would like using tags,
 - The count was `above`, `above or equal to`, `below`, or `below or equal to`
 - the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, or larger. Additionally, you can use `custom` to set a value between 5 minutes and 24 hours.
 
+Process Count in this case, refers to the number of all matching processes that were alive during the time interval.
+
 Use thresholds to set a numeric value for triggering an alert. Datadog has two types of notifications (alert and warning). Live Process Monitors recover automatically based on the alert or warning threshold.
 
 #### Best practices for timeframe selection
 
 Datadog recommends to pick the largest timeframe that you can support. This allows for additional fault tolerance in the case where an Agent becomes unavailable or encounters network issues.
 
-Live Process Monitors uses a [rolling time](https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-window) windows to evaluate process count, e.g. the monitor triggers when the alerting condition is met in the past X minutes. It’s recommended to not use evaluation windows shorter than 5 minutes to prevent the monitor from triggering if there is any sporadic network disruption between the Process Agent and Datadog.
+Live Process Monitors uses a [rolling time](https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-window) window to evaluate process count. In other words, every minute, the monitor checks the past X minutes and triggers if the alerting condition is met. Using evaluation windows shorter than 5 minutes is not recommended in case there is any sporadic network disruption between the Process Agent and Datadog.
```

Comment:
```suggestion
Live Process Monitors uses a [rolling time window](https://docs.datadoghq.com/monitors/configuration/?tab=thresholdalert#evaluation-window) to evaluate process count. In other words, every minute, the monitor checks the past X minutes and triggers if the alerting condition is met. Using evaluation windows shorter than 5 minutes is not recommended in case there is any sporadic network disruption between the Process Agent and Datadog.
```